### DISCUSSION TITLE
Feature Spec: Adding Spring Cloud Azure in App Configuration's MS docs

### DISCUSSION THREAD

**Discussion by chenrujun** [Microsoft/spring-cloud-azure#978] [Ideas]
# Adding Spring Cloud Azure in App Configuration's MS docs

Link to App Configuration's MS docs: https://docs.microsoft.com/en-us/azure/azure-app-configuration/

### Card: Configuration SDKs and samples
![image](https://user-images.githubusercontent.com/13167207/178432569-a51e3e0d-7521-4f43-8b81-bc825d76b8c3.png)

1. Add entry to `Spring Cloud Azure` samples
Text: `Spring Cloud Azure Samples`.
Link: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_v4.3.0/appconfiguration

2. Update `Spring Boot provider`
Current text: `Spring Boot provider`
New text: `Spring Cloud Azure App Configuration`
Current link: https://github.com/microsoft/spring-cloud-azure/tree/master/
New link: https://microsoft.github.io/spring-cloud-azure/docs/azure-app-configuration/2.7.0/reference/html/index.html
Current position: Above `Azure SDK for .NET`.
New position: Under `Azure SDK for Java`.

### Quickstarts
Nothing needs to change. 

### Tutorials
Nothing needs to change. 

### Samples
![image](https://user-images.githubusercontent.com/13167207/178457506-e2d5ced6-f321-4326-b35e-fd38d6c6d638.png)
1. Add entry to `Spring Cloud Azure` samples
Text: `Spring Cloud Azure Samples`.
Link: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_v4.3.0/appconfiguration

### Reference
![image](https://user-images.githubusercontent.com/13167207/178457654-bd7af922-ddf1-424b-af0f-c5a8035f2606.png)
1. Update `Spring Boot provider`
Current text: `Spring Boot provider`
New text: `Spring Cloud Azure App Configuration`
Current link: https://microsoft.github.io/spring-cloud-azure/docs/azure-app-configuration/2.7.0/reference/html/index.html
New link: https://microsoft.github.io/spring-cloud-azure/docs/azure-app-configuration/2.7.0/reference/html/index.html
Current position: Above `Azure SDK for .NET`.
New position: Under `Azure SDK for Java`.

**Comment by chenrujun** [comment]
Notes after first review meeting:

1. Landing page / Configuration SDKs and samples
1.1. Samples:
  1.1.1. No need to add a new entry about "Spring Cloud Azure Samples", just add entry same to "Samples on GitHub" in the TOC, "Samples on GitHub" will have link to "Spring Cloud Azure samples".
1.2. Spring Boot provider
  1.2.1. Text: **(Matt) Need discuss with PM. Options about text: Spring support / Spring Cloud support / Java Spring supprt / ...**
  1.2.2. Link: **(Matt) Need to update.** 
  1.2.3. Position: No need to udpate, because it's ordered by implemenation level, not order by language.

2. TOC
2.1. Samples
  No need to change, because "Samples on GitHub" will have link to "Spring Cloud Azure samples".
2.2. Reference / Java Spring provider.
  2.2.1. Text: (Matt) need to discuss with PM.
  2.2.2. Link: No need to update.
  2.2.3. Position: No need to udpate. Reason is same to "1. Landing page".

**Comment by saragluna** [comment]
1.2.1. Text: (Matt) Need discuss with PM. Options about text: Spring support / Spring Cloud support / Java Spring supprt / ...
--> Matt discussed with the PMs, and they tend to use `Java Spring provider`
1.2.2 Link:
--> https://go.microsoft.com/fwlink/?linkid=2180917

**Reply by chenrujun** [reply]
Thank you for your information.
I'll close this PR: https://github.com/MicrosoftDocs/azure-docs-pr/pull/203989
